app/status/views.py

```diff
@@ -6,4 +7,11 @@
 
 @status.route('/_status')
 def status_no_db():
-    return jsonify(status="ok", version=utils.get_version_label())
+
+    try:
+        return jsonify(status="ok", app_version=utils.get_version_label(),
+                       db_version=utils.get_db_version())
+
+    except SQLAlchemyError:
+        current_app.logger.exception('Cannot connect to database.')
+        return jsonify(status="error", message="Database is down"), 500
```

Comment:
I think the `_status` endpoints on all apps should give any information that could be useful in debugging issues.  If we have the code version to hand then we should include it.  e.g. What if we already know the DB is down (because we're trying to fix it) - it would still be useful to be able to easily check which version of the code is deployed for any particular app.